### PR TITLE
Added functionality for custom callsigns

### PR DESCRIPTION
**Pull request by BonesCrap**
**What does this PR do?**
- Added option for custom callsigns
- To set callsign use [unit, callsign] call orbis_atc_fnc_setCustomCallsign
- If no callsign is set, it defaults to group identifier

**Notice to other developers**
- Changes were tested in SP, not in MP. However it should work in MP
- Edit: When testing consider issue #52 , it may cause problems.

### CODE REVIEW

File: addons/atc/functions/fnc_setCustomCallsign.sqf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include "script_component.hpp"
+
+private _vehicle = _this select 0;
+private _customCallsign = _this select 1;
+
+_vehicle setVariable [QGVAR(customCallsign), _customCallsign, true];
```
Comment (mgkid3310) [inline addons/atc/functions/fnc_setCustomCallsign.sqf]: Adding an empty line at the end would be nice

**Comment by mgkid3310** [comment]
Looks good, will be merged as soon as the comment is applied.

ps. Feel free to add your ID on README.md

File: addons/atc/functions/fnc_setCustomCallsign.sqf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#include "script_component.hpp"
+
+private _vehicle = _this select 0;
```
Comment (synixebrett) [inline addons/atc/functions/fnc_setCustomCallsign.sqf]: use `params` instead of select

**Comment by BonesCrap** [comment]
Done that, thanks for the feedback!

**Comment by mgkid3310** [comment]
PR have been merged, thanks for the contribution!

**Pull request merged**